Source/WebCore/Modules/webcodecs/WebCodecsAudioEncoder.cpp

```diff
@@ -125,6 +125,7 @@ static ExceptionOr<AudioEncoder::Config> createAudioEncoderConfig(const WebCodec
 
 ExceptionOr<void> WebCodecsAudioEncoder::configure(ScriptExecutionContext&, WebCodecsAudioEncoderConfig&& config)
 {
+    Ref pendingActivityKeepingJSWrapperAlive = makePendingActivity(*this);
```

Comment:
Why is it needed to take a pending activity here, given we call synchronously queueControlMessageAndProcess below that will take a pending activity itself?

Reply:
You're right, this one is unnecessary.